app/assets/stylesheets/custom.scss

```diff
@@ -173,4 +173,13 @@ input {
         padding: 10px 0;
         border-bottom: 1px solid $gray-lighter;
     }
+}
+
+.checkbox {
```

Comment:
Scss are like the ruby files where its also Tab Spaces: 2

Reply:
This is not resolved, please change the spacing

Reply:
Hi shania. I've fixed this now :)